tests/suites/host_test.function

```diff
@@ -418,14 +418,18 @@ static void write_outcome_entry( FILE *outcome_file,
  *
  * \param outcome_file  The file to write to.
  *                      If this is \c NULL, this function does nothing.
- * \param unmet_dep_count       The number of unmet dependencies.
- * \param unmet_dependencies    The array of unmet dependencies.
+ * \param unmet_dep_count            The number of unmet dependencies.
+ * \param unmet_dependencies         The array of unmet dependencies.
+ * \param missing_unmet_dependencies Not zero if we were not able to note
```

Comment:
Could we tune the sentence to avoid the double negation? Eg "Non-zero if there was a problem reporting unmet dependencies, 0 otherwise" or "0 if all the unmet dependencies could be reported, non-zero otherwise."

Reply:
Yes that's better, I will change that, thanks.

Reply:
Changed to "Non-zero if there was a problem tracking all unmet dependencies, 0 otherwise."